<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caculator_guiClass</class>
 <widget class="QMainWindow" name="caculator_guiClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1627</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>caculator_gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>1061</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="clearA">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randomA">
         <property name="text">
          <string>随机</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="MatrixA">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入A矩阵&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="add">
         <property name="text">
          <string>A + B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sub">
         <property name="text">
          <string>A - B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="multiply">
         <property name="text">
          <string>A * B</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exchange">
         <property name="text">
          <string>→
←</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="MatrixB">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入B矩阵&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="clearB">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randomB">
         <property name="text">
          <string>随机</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="output_test_browser">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>861</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutA_2">
     <item row="3" column="1">
      <widget class="QPushButton" name="charactorValueA">
       <property name="text">
        <string>特征值</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="charactorVectorA">
       <property name="text">
        <string>特征向量</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="timeA">
         <property name="text">
          <string>幂</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TimeA"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="tranA">
       <property name="text">
        <string>转置矩阵</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="mulityNumA">
         <property name="text">
          <string>数乘</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NumA"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="detA">
       <property name="text">
        <string>行列式</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="trA">
       <property name="text">
        <string>求秩</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="simplestA">
       <property name="text">
        <string>三角矩阵</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="invA">
       <property name="text">
        <string>逆矩阵</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>对角矩阵</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutA">
     <item row="1" column="1">
      <widget class="QPushButton" name="trB">
       <property name="text">
        <string>求秩</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="timeB">
         <property name="text">
          <string>幂</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_TimeB"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="tranB">
       <property name="text">
        <string>转置矩阵</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="invB">
       <property name="text">
        <string>逆矩阵</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="simplestB">
       <property name="text">
        <string>三角矩阵</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="charactorValueB">
       <property name="text">
        <string>特征值</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="charactorVectorB">
       <property name="text">
        <string>特征向量</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="detB">
       <property name="text">
        <string>行列式</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="mulityNumB">
         <property name="text">
          <string>数乘</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NumB"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="dialogB">
       <property name="text">
        <string>对角矩阵</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>640</y>
      <width>222</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QRadioButton" name="use_float">
       <property name="text">
        <string>用小数表示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>小数位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="decimals_num"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>1061</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>矩阵A</string>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>矩阵B</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="caculator_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>add</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_add_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sub</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_subtract_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multiply</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_multiply_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exchange</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_swap_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_randomA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_randomB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1328</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>970</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_clearA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_clearB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1328</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>1089</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_detA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_detB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1026</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>invA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_invA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>invB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_invB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1219</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_tranA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tranB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_tranB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1026</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_rankA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_rankB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1219</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simplestA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_getSimplestA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simplestB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_getSimplestB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1219</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>charactorVectorA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_FeatureVectorA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>charactorVectorB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_FeatureVectorB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1026</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>charactorValueA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_FeatureValueA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>charactorValueB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_FeatureValueB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1219</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mulityNumA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_NumMulA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mulityNumB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_NumMulB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>934</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeA</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_timeA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>681</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_timeB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1127</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>1128</x>
     <y>684</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_float</sender>
   <signal>toggled(bool)</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>change_state_use_float()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>706</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decimals_num</sender>
   <signal>valueChanged(int)</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>change_float_digit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>738</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_dialog_matrixA_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogB</sender>
   <signal>clicked()</signal>
   <receiver>caculator_guiClass</receiver>
   <slot>push_dialog_matrixB_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>955</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>1410</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>push_add_button()</slot>
  <slot>push_subtract_button()</slot>
  <slot>push_multiply_button()</slot>
  <slot>push_swap_button()</slot>
  <slot>push_randomA_button()</slot>
  <slot>push_randomB_button()</slot>
  <slot>push_clearA_button()</slot>
  <slot>push_clearB_button()</slot>
  <slot>push_detA_button()</slot>
  <slot>push_detB_button()</slot>
  <slot>push_invA_button()</slot>
  <slot>push_invB_button()</slot>
  <slot>push_tranA_button()</slot>
  <slot>push_tranB_button()</slot>
  <slot>push_rankA_button()</slot>
  <slot>push_rankB_button()</slot>
  <slot>push_getSimplestA_button()</slot>
  <slot>push_getSimplestB_button()</slot>
  <slot>push_FeatureValueA_button()</slot>
  <slot>push_FeatureValueB_button()</slot>
  <slot>push_FeatureVectorA_button()</slot>
  <slot>push_FeatureVectorB_button()</slot>
  <slot>push_NumMulA_button()</slot>
  <slot>push_NumMulB_button()</slot>
  <slot>push_timeA_button()</slot>
  <slot>push_timeB_button()</slot>
  <slot>change_state_use_float()</slot>
  <slot>change_float_digit()</slot>
  <slot>push_dialog_matrixA_button()</slot>
  <slot>push_dialog_matrixB_button()</slot>
 </slots>
</ui>
